.travis.yml

```diff
@@ -1,23 +1,9 @@
 language: node_js
 node_js:
+  - "16"
+  - "14"
+  - "12"
   - "10"
-  - "9"
-  - "8"
-  - "6"
-  - "4"
-  - "iojs"
-  - "0.12"
```

Comment:
browserify still supports these; whether node supports them or not is irrelevant.

Removing a version from the matrix is tantamount to dropping support for it, which is a breaking change.

Reply:
Interesting. I was not aware of that. Seeing that the [Travis CI jobs of the current commit on the master](https://travis-ci.org/github/browserify/browserify/builds/764377379) failed for Node.js 4 and older I just assumed they were not supported by browserify anymore.

I'll re-add those old versions then. Not fixing any tests that fail on those old versions, though.